shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -85,6 +85,16 @@ public void showSystemOverlays(@NonNull List<PlatformChannel.SystemUiOverlay> ov
           setSystemChromeEnabledSystemUIOverlays(overlays);
         }
 
+        @Override
+        public void showSystemUiMode(@NonNull PlatformChannel.SystemUiMode mode) {
```

Comment:
I'm less sure how these are tested...

Maybe @gaaclarke might have some pointers? I'm just going off of git history and seeing that Aaron has helped Justin with his Clipboard PR in the past, but please redirect to someone else more familiar with engine stuff if there's someone else who can take a look!

Reply:
Yeah, same